### DISCUSSION TITLE
Where is the code for the react-hook-form website housed?

### DISCUSSION THREAD

**Discussion by dili021** [react-hook-form/react-hook-form#8521] [General] [answered]
And where that is also open to contributions? (typo fixes etc.)

**Comment by PaulGBarbu** [comment, marked as answer]
The Website seems to be hosted on Vercel Now, if that's what you mean (scroll all the way down on the homepage)

The repository for the react-hook-form can be found [here](https://github.com/react-hook-form/documentation).

**Reply by dili021** [reply]
Thanks!
